Packaging.props

```diff
@@ -22,6 +22,7 @@
     <!-- by default all packages will use the same version which revs with respect to product version -->
     <PackageVersion Condition="'$(PackageVersion)' == ''">4.4.0</PackageVersion>
     <SkipValidatePackageTargetFramework>true</SkipValidatePackageTargetFramework>
+    <LicenseUrl>https://github.com/dotnet/corefx/blob/master/LICENSE</LicenseUrl>    
```

Comment:
we have a discussion regarding this and we are waiting our PM's to advise. if they advised with changing it I'll go and change this in all repos we did the change in.